django/thunderstore/security/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from thunderstore.security import views
+
+urlpatterns = [
+    path("webhooks/github/ss/", views.secret_scanning_endpoint, name="packages.list"),
```

Comment:
Some notes:
- Path will need to be changed, and could be a bit more descriptive too. Moreover it's not technically a webhook, just an API endpoint GitHub can use.
- How about using `secret-scanning/validate/` for the path? `github` is already included in the core URL pattern end as per the suggested changes there.
- `name="packages.list"` is definitely off here, how about `secret-scanning.validate`?